[core] Batch small changes (#21553)
mui-org/material-ui: docs/src/pages/guides/migration-v4/migration-v4.md

### Typography

> - Replace `srOnly` prop support with a style util to not duplicate with the capabilities of the system:

Reviewer: This needs rewording.

Author: ```suggestion
- Replace `srOnly` prop to not duplicate with the capabilities of the "system":
```

Reviewer 2: ```suggestion
- Replace `srOnly` prop so as to not duplicate the capabilities of [System](https://material-ui.com/system/basics/):
```
